crates/bevy_gizmos/src/lib.rs

```diff
@@ -246,29 +249,46 @@ impl AppGizmoBuilder for App {
 
         // These handles are safe to mutate in any order
         self.allow_ambiguous_resource::<LineGizmoHandles>();
-
-        self.init_resource::<GizmoStorage<Config, ()>>()
-            .init_resource::<GizmoStorage<Config, Fixed>>()
-            .init_resource::<GizmoStorage<Config, Swap<Fixed>>>()
-            .add_systems(
-                RunFixedMainLoop,
-                start_gizmo_context::<Config, Fixed>
-                    .in_set(bevy_app::RunFixedMainLoopSystem::BeforeFixedMainLoop),
-            )
-            .add_systems(FixedFirst, clear_gizmo_context::<Config, Fixed>)
-            .add_systems(FixedLast, collect_requested_gizmos::<Config, Fixed>)
-            .add_systems(
-                RunFixedMainLoop,
-                end_gizmo_context::<Config, Fixed>
-                    .in_set(bevy_app::RunFixedMainLoopSystem::AfterFixedMainLoop),
-            )
-            .add_systems(
-                Last,
-                (
-                    propagate_gizmos::<Config, Fixed>.before(UpdateGizmoMeshes),
-                    update_gizmo_meshes::<Config>.in_set(UpdateGizmoMeshes),
-                ),
-            );
+        #[cfg(feature = "fixed_time")]
+        {
+            self.init_resource::<GizmoStorage<Config, ()>>()
```

Comment:
Pull the common stuff out of the `cfg` blocks for clarity and maintainability please :)

Reply:
Makes sense, Will get this done rq :+1: